worksheet_B/README.md

```diff
@@ -0,0 +1,35 @@
+```
+START
+
+password = RANDOMLY PICKED FROM BANK/MATRIX
+guess_attempts = 0                                    //SETS NUMBER OF PLAYER GUESSES AT LOWEST VALUE
+
+WHILE (guess_attempts<3)                              //ALLOWS LOOP TO RUN WHILE THE PLAYER STILL HAS GUESSES REMAINING
+    guessedletter=randomletter                        //MATCHES PLAYER INPUT TO RANDOM WORD CHARACTER
+    characterposition=0                            //SETS CHARACTER READ NUMBER of RANDOM PASSWORD I.E. FIRST LETTER = 0, SECOND = 1
+    similarity=0                                      //SIMILARITY VALUE OF THE TWO WORDS
+    
+     WHILE(characterposition<4)
+        IF (guessedletter=randomletter)               //WHILE LOOP COMPARES THE PLAYER INPUTTED LETTER WITH THE RANDOM PASSWORDS LETTER
```

Comment:
Need to make it clear that guessedletter and randomletter are the (characterposition)th letters of the respective words